src/components/Navbar/Navbar.js

```diff
@@ -1,52 +1,61 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import logo from "../../assets/logo-sm.png";
-import logo_txt from '../../assets/logo-txt.png'
+import logo_txt from "../../assets/logo-txt.png";
```

Comment:
Try barrel imports here. 

Check this https://srinivasankk.com/javascript-barrel/  or if it's still not clear, you can check one of my projects